opml-importer.php

```diff
@@ -50,7 +53,6 @@ function dispatch() {
 ?>
 
 <div class="wrap">
-<?php screen_icon(); ?>
```

Comment:
This plugin [still supports WP 3.0](https://github.com/WordPress/opml-importer/blob/1c9caeeb2116d1bff4d08eaf07ad280644b1d552/readme.txt#L5) - should this function maybe be called conditionally ?